src/main/java/es/tid/pce/pcep/objects/tlvs/SRPolicyCandidatePathPreferenceTLV.java

```diff
@@ -0,0 +1,105 @@
+package es.tid.pce.pcep.objects.tlvs;
+
+import java.util.Objects;
+
+import es.tid.pce.pcep.objects.MalformedPCEPObjectException;
+import es.tid.protocol.commons.ByteHandler;
+
```

Comment:
Add header with name of the TLV, reference (https://datatracker.ietf.org/doc/draft-ietf-pce-segment-routing-policy-cp/04/), and author